internal/commands/pull/command.go

```diff
@@ -60,33 +62,58 @@ func (cmd *Command) Handler(profile *user.Profile, ui terminal.UI, clients cli.C
 		return err
 	}
 
-	pathTarget, zipPkg, err := cmd.doExport(profile, clients.Realm, app.GroupID, app.ID)
-	if err != nil {
-		return err
+	// TODO(REALMC-9462): maybe make this less hacky (pass app in to resolveClientTemplates perhaps?)
+	var clientZipPkgs map[string]*zip.Reader
+	if app.TemplateID != "" {
+		clientZipPkgs, err = cmd.inputs.resolveClientTemplates(ui, clients.Realm, app.GroupID, app.ID)
+		if err != nil {
+			return err
+		}
 	}
 
-	pathRelative, err := filepath.Rel(profile.WorkingDirectory, pathTarget)
+	pathProject, zipPkg, err := cmd.doExport(profile, clients.Realm, app.GroupID, app.ID)
 	if err != nil {
 		return err
 	}
 
-	proceed, err := checkAppDestination(ui, pathTarget)
+	var pathFrontend string
```

Comment:
[meganit] we probably don't need to do this pathing logic until after the first `checkPathDestination` (maybe even after the `pathRelative` logic too, just before `if dryRun`)